src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -40,6 +44,16 @@
     private volatile int storageCheckInterval = Integer.MAX_VALUE;
     private final AtomicBoolean resetQuota = new AtomicBoolean(false);
     private final StorageChecker storageChecker = new StorageChecker();
+    private final Gauge limitBytes = KafkaYammerMetrics.defaultRegistry().newGauge(StaticQuotaCallback.class, "LimitBytes", new Gauge<Long>() {
+        public Long value() {
+            return storageQuotaSoft;
+        }
+    });
+    private final Gauge usedBytes = KafkaYammerMetrics.defaultRegistry().newGauge(StaticQuotaCallback.class, "UsedBytes", new Gauge<Long>() {
+        public Long value() {
+            return storageUsed.get();
```

Comment:
This metric is a sum of used storage across all log dirs. Its name should indicate that and allow for per-log-dir metrics in the future.